ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -78,7 +82,14 @@ private class PackageService(
   private class StateCache private () {
     // volatile, reading threads don't need synchronization
     @volatile private var _state: State =
-      State(Set.empty, TemplateIdMap.Empty, TemplateIdMap.Empty, Map.empty, Map.empty, Map.empty)
+      State(
+        Set.empty,
+        TemplateIdMap.Empty,
+        TemplateIdMap.Empty,
+        Map.empty,
+        Map.empty,
+        Map.empty,
```

Comment:
Using field names would make that a bit more readable.

Reply:
I would normally agree, if there was any chance that mixing up arguments would yield the wrong result, but that isn't the case here. We do use keyword arguments where all the arguments aren't the same.

Reply:
...or, we don't use them where the arguments aren't the same! I'll add keyword arguments to the other call, thanks for the impetus.

Reply:
fair enough

Reply:
b129b0e1f09